quarto-formats.qmd

```diff
@@ -1,49 +1,49 @@
-# Quarto formats {#sec-quarto-formats}
+# Formatos para Quarto {#sec-quarto-formats}
 
 ```{r}
 #| echo: false
 
 source("_common.R")
 ```
 
-## Introduction
+## Introdução
 
-So far, you've seen Quarto used to produce HTML documents.
-This chapter gives a brief overview of some of the many other types of output you can produce with Quarto.
+Até agora você viu o Quarto sendo usado para produzir documentos HTML.
+Este capítulo dá uma visão geral de alguns dos muitos outros tipos de saída (*output*) que você pode produzir com Quarto.
 
-There are two ways to set the output of a document:
+Há duas formas de configurar a saída do documento:
 
-1.  Permanently, by modifying the YAML header:
+1.  Permanentemente, modificando o cabeçalho YAML:
 
     ``` yaml
-    title: "Diamond sizes"
+    title: "Tamanhos de diamante"
     format: html
     ```
 
-2.  Transiently, by calling `quarto::quarto_render()` by hand:
+2.  Pontualmente, executando `quarto::quarto_render()` manualmente:
 
     ```{r}
     #| eval: false
 
     quarto::quarto_render("diamond-sizes.qmd", output_format = "docx")
     ```
 
-    This is useful if you want to programmatically produce multiple types of output since the `output_format` argument can also take a list of values.
+    Isso é útil quando se deseja programaticamente produzir múltiplos tipos de saída ja que o argumento `output_format` também pode receber uma lista de valores.
 
     ```{r}
     #| eval: false
 
     quarto::quarto_render("diamond-sizes.qmd", output_format = c("docx", "pdf"))
     ```
 
-## Output options
+## Opções de saída
 
-Quarto offers a wide range of output formats.
-You can find the complete list at <https://quarto.org/docs/output-formats/all-formats.html>.
-Many formats share some output options (e.g., `toc: true` for including a table of contents), but others have options that are format specific (e.g., `code-fold: true` collapses code chunks into a `<details>` tag for HTML output so the user can display it on demand, it's not applicable in a PDF or Word document).
+O Quarto oferece uma ampla variedade de formatos de saída.
+É possível achar a lista completa em <https://quarto.org/docs/output-formats/all-formats.html>.
+Muitos formatos compartilham configurações de seu resultado (por exemplo, `toc: true` para incluir um índice), mas outros têm configurações que são específicas do formato (por exemplo, `code-fold: true` colapsa os blocos de código em um tag HTML `<details>` de modo que o usuário possa exibí-lo quando quiser, e não é aplicável a um documento PDF ou Word).
```

Comment:
Teoricamente, `sumário` é o que aparece no começo de um livro, e `índice` é aquela seção no final de livros acadêmicos que tem várias palavras-chave e a página que aparecem. Neste caso, creio que `sumário` seja o mais correto para `table of contents`, a não ser que o Quarto utilize assim.
```suggestion
Muitos formatos compartilham configurações de seu resultado (por exemplo, `toc: true` para incluir um sumário), mas outros têm configurações que são específicas do formato (por exemplo, `code-fold: true` colapsa os blocos de código em uma *tag* HTML `<details>` de modo que o usuário possa exibí-lo quando quiser, e não é aplicável a um documento PDF ou Word).
```